sematic/plugins/storage/local_storage.py

```diff
@@ -1,27 +1,25 @@
 # Standard Library
 import logging
 import os
-from typing import Optional
+from http import HTTPStatus
+from typing import Dict, Optional
 
 # Third-party
 import flask
 
 # Sematic
 from sematic.abstract_plugin import AbstractPlugin, PluginVersion
 from sematic.api.app import sematic_api
-from sematic.api.endpoints.auth import authenticate
+from sematic.api.endpoints.auth import API_KEY_HEADER, authenticate
+from sematic.api.endpoints.request_parameters import jsonify_error
 from sematic.config.config import get_config
+from sematic.config.server_settings import get_api_address
 from sematic.db.models.user import User
-from sematic.plugins.abstract_storage import (
-    AbstractStorage,
-    NoSuchStorageKeyError,
-    PayloadType,
-    ReadPayload,
-)
+from sematic.plugins.abstract_storage import AbstractStorage, Location
 
 logger = logging.getLogger(__name__)
 
-_PLUGIN_VERSION = (0, 1, 0)
+_PLUGIN_VERSION = (0, 2, 0)
```

Comment:
Remind me: are we using these versions yet to auto-migrate settings file formats for specific plugins? If so, does this PR need to add some settings file format migration logic?

Reply:
If no edits need to be made to the settings file itself, then the version can be kept the same.